queues.md

```diff
@@ -62,7 +62,7 @@ While building your web application, you may have some tasks, such as parsing an
 
 Laravel queues provide a unified queueing API across a variety of different queue backends, such as [Amazon SQS](https://aws.amazon.com/sqs/), [Redis](https://redis.io), or even a relational database.
 
-Laravel's queue configuration options are stored in your application's `config/queue.php` configuration file. In this file, you will find connection configurations for each of the queue drivers that are included with the framework, including the database, [Amazon SQS](https://aws.amazon.com/sqs/), [Redis](https://redis.io), and [Beanstalkd](https://beanstalkd.github.io/) drivers, as well as a synchronous driver that will execute jobs immediately (for use during local development). A `null` queue driver is also included which discards queued jobs.
+Laravel's queue configuration options are stored in your application's `config/queue.php` configuration file. In this file, you will find connection configurations for each of the queue drivers that are included with the framework, including the database, [Amazon SQS](https://aws.amazon.com/sqs/), [Redis](https://redis.io), [MongoDB](https://www.mongodb.com/), and [Beanstalkd](https://beanstalkd.github.io/) drivers, as well as a synchronous driver that will execute jobs immediately (for use during local development). A `null` queue driver is also included which discards queued jobs.
```

Comment:
This is adding MongoDB to a list of "queue drivers that are included with the framework", which I don't think is appropriate. Consider adding a separate sentence before introduction of the `null` driver. Similar to what you did in other pages, you can note that the integration is provided by the `mongodb/laravel` package.

I'll defer to @bisht2050 if it makes sense to link to the MongoDB homepage here.

Reply:
Agreed, we should have a separate line to avoid misleading readers, and link to our [queues docs](https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/queues/)